### PR TITLE
Copyright: various fixes

### PR DESCRIPTION
**Pull request by na-na-hi**
Fixes some inaccurate descriptions.

### CODE REVIEW

File: Copyright [hunk 1]
```diff
@@ -39,8 +39,7 @@ quite central:
 - Linux X11 video output
 - BSD audio output via OSS
 - NVIDIA/Linux hardware decoding (vdpau, although nvdec usually works)
-- Linux TV input
```
Comment (sfan5) [inline Copyright:42]: this is stream/stream_dvb.c, which still exists

Comment (sfan5) [reply Copyright:42]: oh, I see

**Review by kasper93 — APPROVED** [review]
I'm not a licence expert, but the wording change make sense to me and are clear to understand.

**Review by Dudemanguy — APPROVED** [review]
I mean I don't really know how this works either. Sure I guess.

**Pull request merged**